PopulateTools/gobierto
# Enhance sections to implement site navigation

Improvement

### What does this PR do?

This PR enhances sections to create a site navigation section which can include references to:

- sections
- modules

### How should this be manually tested?

1. Create a section with slug `global-navigation`
2. Add modules and sections as section items
3. Check in the front the new navigation menu (it should overwrite the default one)
